### ISSUE TITLE
Unprompted ChromeOS interaction 

### ISSUE THREAD

**Issue by boognish-rising** [tytydraco/LADB#86]
I thought that I had made mention of this previously but I don't see any mention of it in any issues or even comments so I could be wrong and if I'm not I don't know where I made mention of it but many times now when I've gone to use LADB instead of it trying to initiate wireless debugging on my phone (Google Pixel 6 Pro Android 13) It seems to somehow be communicating with my Pixelbook, which is obviously a completely separate device. I'm at a loss for why this is happening. I do have the Pixelbook connected to my Pixel via the Phone Hub feature/functionality of ChromeOS, but that's just connecting via Bluetooth as far as I know but even if it was connecting via a Wi-Fi connection or even over the network, I don't see why that would cause this to happen. Am I missing something or is this completely weird behavior? 

![PXL_20221012_173330031.jpg](https://user-images.githubusercontent.com/76668060/195411621-cd3e0188-0534-4c78-a6fd-0f258d10df72.jpg)

![Screenshot_20221012-133313.png](https://user-images.githubusercontent.com/76668060/195411693-683dc5dd-5c35-409f-acfe-28a3586f8fd8.png)

**Comment by teou1** [comment]
I'd say it sounds like google bug with chrome os and the phone hub, which is quite buggy and does not always connect on my Chomeos 105 and pixel4a A13 setup. However, i have not enabled adb on the chromebook, i guess that's the difference.

If i were you i'd try 2 things: manually starting witeless debug on the phone before running  ladb, and maybe turning off bluetooth or/and wifi on the Chromebook until ladb connects.

**Comment by ShadowFlare** [comment]
Sounds related to #61 that I opened.

**Comment by boognish-rising** [comment]
You're absolutely right. Sounds identical, even. My fault. I did a search but I searched for "Chromebook"  and "ChromeOS"  and didn't think it was an issue that extended to network devices beyond Chromebooks/Chrome OS specifically. 

That said, feel free to either close this as a duplicate or merge it or whatever you think is necessary or more conducive to keeping things tidy around here @tytydraco 

Apologies for missing #61 that @ShadowFlare mentioned/started...  I'll keep an eye on that for progress/new developments, etc
